### PR TITLE
new emp_act for experimental laser guns

### PR DESCRIPTION
**Pull request by PKPenguin321**
kaboom

### CODE REVIEW

File: code/modules/projectiles/guns/energy/nuclear.dm [hunk 1]
```diff
@@ -79,3 +79,9 @@
 	ammo_x_offset = 1
 	ammo_type = list(/obj/item/ammo_casing/energy/electrode, /obj/item/ammo_casing/energy/laser, /obj/item/ammo_casing/energy/disabler)
 	selfcharge = 1
+
+/obj/item/weapon/gun/energy/gun/nuclear/emp_act(severity)
+	var/gun = src.loc
+	gun.visible_message("<span class='userdanger'>\The [src]'s nuclear reactor overloads!</span>")
+	explosion(src.loc,1,2,4,flame_range = 2)
```
Comment (PKPenguin321) [inline code/modules/projectiles/guns/energy/nuclear.dm]: FUCk me

**Comment by kevinz000** [comment]
I DED PLS NERF

**Comment by GunHog** [comment]
An explosion is a bit nuts, why not have it blast strong radiation or something?

**Comment by kevinz000** [comment]
THIS IS LITERALLY A SALT PR AIMED AT HIM GETTING COMPLETELY DESTROYED IN NUKEOPS.

**Comment by Jordie0608** [comment]
Where did all the code for the nuclear gun go?

**Comment by Jordie0608** [comment]
So it seems #17576 deleted the unique functionality for the nuclear gun when removing reliability; How about you change this PR to just re-add that?

**Comment by PKPenguin321** [comment]
the what now

**Comment by PKPenguin321** [comment]
>Gibbing the holder is FAR TOO MUCH.

ruining Pripyat was a bitch much too but that's what happens when a nuclear reactor gets fucked with

**Comment by Jordie0608** [comment]
I did it myself #19915

**Comment by optimumtact** [comment]
the emperor wears no cloths.

**Pull request closed without merging**